### ISSUE TITLE
Block Comments Accessibility: make the "n more replies" text clickable

### ISSUE THREAD

**Issue by jeffpaul** [WordPress/gutenberg#71680]
### Description

From @t-hamano in https://github.com/WordPress/gutenberg/issues/71249#issue-3333938703:

> The comment sidebar shows `{count} more replies..` text when there are a lot of replies. The hidden comments can be expanded by clicking with a mouse, but can not be expanded because there is no focusable button to expand them.

From @joedolson in https://github.com/WordPress/gutenberg/issues/71249#issuecomment-3201284909:

> You can expand the list by picking the options menu; clicking on anything within the comment box triggers the expansion. However, there's absolutely nothing intuitive for a user about needing to click "Select an action" in order to read the rest of the comments. Making the "n more replies" text clickable would make this much more intuitive.

### Step-by-step reproduction instructions

.

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure
